tests/pass/tls/win_tls_callback.rs

```diff
@@ -5,6 +5,8 @@
 // that's not what this test is about.
 //@compile-flags: -Zmiri-ignore-leaks
 
+//@only-miri: FIXME: fatal runtime error: thread::set_current should only be called once per thread
```

Comment:
Oh lol.^^

We probably can't use `eprintln!` so early during the setup. So maybe directly use the underlying system functions instead?